js/newUser.js

```diff
@@ -20,10 +20,43 @@ form.addEventListener('submit', (e) => {
         const password = document.getElementById('password').value;
         alert(`User create succesfully. The date is Username: ${username}, Email: ${email} and Password: ${password}`);
 
+        saveToLocalStorage()
+
         window.history.back();
-        }
+    }
     catch (error) {
         alert("You made a mistake. Please try again.");
         console.log(error);
     }
-    })
+})
+
+function saveToLocalStorage() {
+    let newEmail = document.getElementById('email').value
+    let newUsername = document.getElementById('username').value
+    let newPassword = document.getElementById('password').value
+
+    if (localStorage.getItem('email') == null) {
+        localStorage.setItem('email', '[]' )
+    }
+
+    if (localStorage.getItem('username') == null) {
+        localStorage.setItem('username', '[]' )
+    } 
+
+    if (localStorage.getItem('password') == null) {
+        localStorage.setItem('password', '[]' )
+    }
+
+    let oldEmail = JSON.parse(localStorage.getItem('email'))
+    oldEmail.push(newEmail)
+
+    let oldUsername = JSON.parse(localStorage.getItem('username'))
+    oldUsername.push(newUsername)
+
+    let oldPassword = JSON.parse(localStorage.getItem('password'))
+    oldPassword.push(newPassword)
```

Comment:
NEVER SAVE PASSWORD IN PLAIN TEXT

Reply:
look for encryption algorithms to save the encrypted password in local storage. The encryption algorithm should be 1:1.